pwa/src/components/dashboardCard/DashboardCard.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
 import * as styles from "./DashboardCard.module.css";
-import { Button, Link } from "@gemeente-denhaag/components-react";
+import { Link } from "@gemeente-denhaag/components-react";
```

Comment:
Please check your imports, there are some unused ones. Anything that's not used, can be removed.